plugins/modules/redfish_storage_volume.py

```diff
@@ -260,6 +304,12 @@
 storage_collection_map = {}
 CHANGES_FOUND = "Changes found to be applied."
 NO_CHANGES_FOUND = "No changes found to be applied."
+RAID_TYPE_NOT_SUPPORTED_MSG = "RAID Type {raid_type} is not supported."
+volume_type_map = {"NonRedundant": "RAID0",
+                   "Mirrored": "RAID1",
+                   "StripedWithParity": "RAID5",
+                   "SpannedMirrors": "RAID10",
+                   "SpannedStripesWithParity": "RAID50"}
```

Comment:
At which line or snippet, we are handling RAID6 and RAID60?

Reply:
Riad6 and Raid60 doesnt have this other names supported. So no mapping is done here. in function volume_payload, we are mapping all the values.